src/Compilers/CSharp/Test/Symbol/Symbols/RequiredMembersTests.cs

```diff
@@ -1242,63 +1242,186 @@ public void RefReturningProperties()
 class C
 {
     private int i;
-    public required ref int Prop => ref i;
+    public required ref int Prop1 => ref i;
```

Comment:
Based on this, I think the spec should say "`ref` - when added for ref ~~fields~~ properties".

Reply:
Yeah, I'll send an update.